plugins/snowflake/dbt/include/snowflake/macros/adapters.sql

```diff
@@ -15,6 +15,23 @@
   );
 {% endmacro %}
 
+{% macro snowflake__create_or_replace_table_as(relation, sql) -%}
```

Comment:
yeah, just add `or replace` to line 9 above and i think we can delete this whole macro